src/main/java/com/google/sps/data/DatastoreEntityToObjectConverter.java

```diff
@@ -0,0 +1,83 @@
+package com.google.sps.data;
+
+import java.util.List;
+import java.util.ArrayList;
+import com.google.sps.data.Comment;
+import com.google.sps.data.Constants;
+import com.google.sps.data.Post;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Key; 
+import com.google.appengine.api.datastore.KeyFactory;
+
+/**
+ * Converts entities from datastore into objects 
+ */  
+public class DatastoreEntityToObjectConverter {
+    /**
+     * Converts a representative entity into a representative object 
+     * @param entity of the representative 
+     * @return the representative object 
+     */  
+    protected static Representative convertRepresentative(Entity entity) throws EntityNotFoundException{
```

Comment:
I dont think this class has any subclasses so lets make this package private (ie have no modifier). 

https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html